Commit hashes different between splitsh-lite and git subtree split

I tried to use splitsh-lite to split the monolithic development repositories of Neos CMS into the readonly package repositories, but I end up with different commit hashes compared to the current approach using git subtree split.

Versions used (ubuntu 14.04), splitsh-lite compiled today according to instructions in readme
```
$ git --version
git version 1.9.1
$ go version
go version go1.6 linux/amd64
````

To reproduce:
```
$ git clone https://github.com/neos/neos-development-collection.git neos-dev
$ cd neos-dev
$ splitsh-lite --prefix=TYPO3.Neos --target=refs/heads/split-neos-master-splitsh --origin=origin/master --git '<2.8.0' --scratch
7197 commits created, 9984 commits traversed, in 4.09s
e494289e6d34edec5b35a0add75ce87e73b344c2
$ git subtree split --prefix="TYPO3.Neos" --branch=split-neos-master-subtree origin/master
Created branch 'split-neos-master'
c3034f367ecb1e4f5b4e5a36b6fe15d0be06cabd
$ git rev-list --count split-neos-master-splitsh
7182
$ git rev-list --count split-neos-master-subtree
7182
$ git log --oneline --topo-order split-neos-master-splitsh | tail -n 5
fd5d14c * FLOW3: (Component) Removed support for setter injection via generic "setProperty()" method. * FLOW3: (Component) Setter injection is now also possible via a "inject" method, e.g. injectSomeDependency() (instead of setSomeDependency). All methods whose name starts with "inject" will be included into the autowiring procedure. Therefore dependencies don't need to be configured explicitly in the Components.ini when "inject" methods are used. Resolves #319. * FLOW3: (Component) Added autowiring support for "inject" setter methods. Addresses #319. * Smarty: Added check against multiple definition of "SMARTY_DIR" * Testing, TYPO3, PhoneBookTutorial: Refactored to the new "inject*" methods in the action controllers. Original-Commit-Hash: 14ceca885be55807c8584116b7eeabfa1b710622
cccdefc * FLOW3: (MVC) Improved the directory structure and naming of views. They now have the name of the controller in their own name. Fixes #314. * Updated the views in other packages accordingly. * Fixed whitespace and inline doc issues in classes I came across. Original-Commit-Hash: 8919dc4a43eb1eb0a5756f27b9cf123ef5bbc960
5ad77ca * FLOW3: (Package) Moved the Components.conf file from the Classes/ directory of each package to a new Configuration/ directory and renamed the file to Components.ini. Closes #309. * FLOW3: Updated the documentation accordingly. Original-Commit-Hash: bbae6513443e2297eec210c08e3ddf9aacf8d37c
6cba34a * FLOW3: Fixed whitespaces and inline comments where I came across * FLOW3: (MVC) Added two new exceptions: NoSuchController and InvalidController (for Dispatcher) * FLOW3: (MVC) Added an initial implementation of a router for web requests. Tests are still in the Web Request Builder test case but will be refactored soon. * TYPO3: A bit of cleanup and reviving of the Page controller * Testing: Changed "POST" to "post" in MainLayout.html. That fixes #264, but in general it's too early to look at errors of this kind because the new Testrunner will still change a lot. Original-Commit-Hash: 984b039f4a2a82d1cbee3e89a47c3b43f9752c3c
a558e62 Subversion reorganisation, part 5:  * Moved some directories again to create a structure agreed upon by the RnD committee Original-Commit-Hash: 22915448b2c9da021015c59f680cecde4e071849
$ git log --oneline --topo-order split-neos-master-subtree | tail -n 5
f370041 * FLOW3: (Component) Removed support for setter injection via generic "setProperty()" method. * FLOW3: (Component) Setter injection is now also possible via a "inject" method, e.g. injectSomeDependency() (instead of setSomeDependency). All methods whose name starts with "inject" will be included into the autowiring procedure. Therefore dependencies don't need to be configured explicitly in the Components.ini when "inject" methods are used. Resolves #319. * FLOW3: (Component) Added autowiring support for "inject" setter methods. Addresses #319. * Smarty: Added check against multiple definition of "SMARTY_DIR" * Testing, TYPO3, PhoneBookTutorial: Refactored to the new "inject*" methods in the action controllers. Original-Commit-Hash: 14ceca885be55807c8584116b7eeabfa1b710622
84d63de * FLOW3: (MVC) Improved the directory structure and naming of views. They now have the name of the controller in their own name. Fixes #314. * Updated the views in other packages accordingly. * Fixed whitespace and inline doc issues in classes I came across. Original-Commit-Hash: 8919dc4a43eb1eb0a5756f27b9cf123ef5bbc960
f9aa0c9 * FLOW3: (Package) Moved the Components.conf file from the Classes/ directory of each package to a new Configuration/ directory and renamed the file to Components.ini. Closes #309. * FLOW3: Updated the documentation accordingly. Original-Commit-Hash: bbae6513443e2297eec210c08e3ddf9aacf8d37c
644d9a2 * FLOW3: Fixed whitespaces and inline comments where I came across * FLOW3: (MVC) Added two new exceptions: NoSuchController and InvalidController (for Dispatcher) * FLOW3: (MVC) Added an initial implementation of a router for web requests. Tests are still in the Web Request Builder test case but will be refactored soon. * TYPO3: A bit of cleanup and reviving of the Page controller * Testing: Changed "POST" to "post" in MainLayout.html. That fixes #264, but in general it's too early to look at errors of this kind because the new Testrunner will still change a lot. Original-Commit-Hash: 984b039f4a2a82d1cbee3e89a47c3b43f9752c3c
a558e62 Subversion reorganisation, part 5:  * Moved some directories again to create a structure agreed upon by the RnD committee Original-Commit-Hash: 22915448b2c9da021015c59f680cecde4e071849
```

As you can see in the git log snippets, the first commit is the same, but from the second commit on the hashes differ. I'm not sure what further information to include or how to analyze this problem further. Do you have any idea what could cause this?

--- a/splitter/state.go
+++ b/splitter/state.go
@@ -467,7 +467,7 @@ func (s *state) copyCommit(rev *git.Commit, tree *git.Tree, parents []*git.Commi
 		s.logger.Printf("  copy commit \"%s\" \"%s\" \"%s\"\n", rev.Id().String(), tree.Id().String(), strings.Join(parentStrs, " "))
 	}
 
-	message := rev.Message()
+	message := rev.RawMessage()
 	if s.config.Git == 1 {
 		message = s.legacyMessage(rev)
 	}

